server/views/pages/accessDenied.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - access denied" %}
+{% set pageId = "access-denied-page" %}
+
+{% block content %}
+    <div class="govuk-grid-row govuk-body">
+        <div class="govuk-grid-column-two-thirds">
+            {{ govukWarningText({
+                text: "Access is denied",
+                iconFallbackText: "Warning"
+            }) }}
+
+            <h2 class="govuk-heading-s">Why am I seeing this page?</h2>
```

Comment:
I think Jo has been thinking about content for error pages so this might have discussion but fine for now

Reply:
yes I agree. Its a placeholder, until she sees it.